README.md

```diff
@@ -2,105 +2,145 @@
+## 🚀 Getting Started
 
-### License
-Stride is covered by the [MIT License](LICENSE.md) unless stated otherwise (i.e. for some files that are copied from other projects). You can find the list of third-party projects [here](THIRD%20PARTY.md). Contributors need to sign the following [Contribution License Agreement](https://github.com/dotnet-foundation/.github/blob/main/CLA/dotnetfoundation.yml).
+If you want to **create games using Stride**, please visit our [Get started with Stride](https://doc.stride3d.net/latest/en/manual/get-started/index.html) guide. This guide provides detailed instructions on how to download, install, and begin using Stride to develop your games.
+
+If you are interested in **building the Stride engine from source** or **contributing to its development**, please continue reading this README for instructions on how to build from source and contribute to the project.
+
+If you are interested in both, we recommend starting with the [Get started with Stride](https://doc.stride3d.net/latest/en/manual/get-started/index.html) guide, and then returning here to learn how to build from source and contribute.
+
+## 🤝 Contributing
+
+For questions and general discussions, please join our [Discord server](https://discord.gg/f6aerfE) or participate in [GitHub Discussions](https://github.com/stride3d/stride/discussions).
+
+To report bugs or propose features, please use the [Issues](https://github.com/stride3d/stride/issues) section on GitHub.
 
-### Code of conduct
-Stride being a [.NET Foundation](https://www.dotnetfoundation.org/) project, it has adopted the code of conduct defined by the Contributor Covenant to clarify expected behavior in our community.
-For more information see the [.NET Foundation Code of Conduct](https://dotnetfoundation.org/code-of-conduct). 
+We welcome code contributions via pull requests. Issues tagged with **[`good first issue`](https://github.com/stride3d/stride/labels/good%20first%20issue)** are great starting points for code contributions.
 
-## Documentation
+You can help us translate Stride; check out our [Localization Guide](https://doc.stride3d.net/latest/en/contributors/engine/localization.html).
 
-Find explanations and information about Stride:
-* [Stride Manual](https://doc.stride3d.net/latest/manual/index.html)
-* [Tutorials](https://doc.stride3d.net/latest/en/tutorials/)
-* [API Reference](https://doc.stride3d.net/latest/api/index.html)
-* [Release Notes](https://doc.stride3d.net/latest/ReleaseNotes/index.html)
+### Earn Money by Contributing
 
-## Community
+If you are a developer with solid experience in C#, rendering techniques, or game development, we want to hire you! We have allocated funds 💰 from supporters on [Open Collective](https://opencollective.com/stride3d) and can pay for work on certain projects. [More information is available here](https://doc.stride3d.net/latest/en/contributors/engine/bug-bounties.html).
```

Comment:
Emojis are fine and all, but not in the middle of a sentence. It would be fine for a text message, not so much for a readme I don't think, it kind of feels like we aren't taking this seriously.

Reply:
Sure, it will be removed.

Reply:
Done.